src/custom/pages/Profile/LockedGnoVesting/claimData/rinkeby.json

```diff
@@ -0,0 +1,159 @@
+[
```

Comment:
I think this was talked about while I was away, but wouldn't be better to store the claim indexes in the same repo as the claim files instead of adding them to the Cowswap's repo like was done for the airdrop?
Or we expect the amount to be significantly smaller that this is acceptable?

I'm personally not very found of having them here.